---a/net/minecraft/network/rcon/IServer.java
+++b/net/minecraft/network/rcon/IServer.java
@@ -1,10 +1,6 @@
 package net.minecraft.network.rcon;
 
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-@SideOnly(Side.SERVER)
-public interface IServer
-{
+public interface IServer {
+
     int getIntProperty(String key, int defaultValue);
 
     String getStringProperty(String key, String defaultValue);
@@ -45,4 +45,4 @@
     void logSevere(String msg);
 
     void logDebug(String msg);
-}
+}
